multiqc/modules/disambiguate/disambiguate.py

```diff
@@ -0,0 +1,137 @@
+
+                # Add to data and add data source.
+                self.data[sample] = counts
+                self.add_data_source(summary_file, s_name=sample)
+
+        self.data = self.ignore_samples(self.data)
+
+        if len(self.data) == 0:
+            log.debug("Could not find any data in %s", config.analysis_dir)
+            raise UserWarning
+
+        log.info("Found %d reports", len(self.data))
+
+        self.add_stats_table()
+        self.add_stats_plot()
+        self.write_stats_to_file()
+
+
+    def parse_summary(self, contents):
+        """Parses summary file into a dictionary of counts."""
+
+        lines = contents.strip().split('\n')
+
+        data = {}
+        for row in lines[1:]:
+            split = row.strip().split('\t')
+
+            sample = split[0]
+
+            data[sample] = {
+                'species_a': int(split[1]),
+                'species_b': int(split[2]),
+                'ambiguous': int(split[3])
+            }
+
+        return data
+
+
+    def add_stats_table(self):
+        """Adds stats to general table."""
+
+        totals = {sample: sum(counts.values())
+                  for sample, counts in self.data.items()}
+
+        percentages = {sample: {k: (v / totals[sample]) * 100
+                                for k, v in counts.items()}
+                       for sample, counts in self.data.items()}
+
+        headers = OrderedDict()
+
+        headers['species_a'] = {
+            'title': '% Species a',
+            'description': 'Percentage of reads mapping to species a',
+            'max': 100,
+            'min': 0,
+            'suffix': '%',
+            'scale': 'YlGn'
+        }
+
+        headers['species_b'] = {
+            'title': '% Species b',
+            'description': 'Percentage of reads mapping to species b',
+            'max': 100,
+            'min': 0,
+            'suffix': '%',
+            'scale': 'YlGn'
+        }
+
+        headers['ambiguous'] = {
+            'title': '% Amb.',
+            'description': ('Percentage of ambiguous reads (which map'
+                            ' to both species a and b)'),
+            'max': 100,
+            'min': 0,
+            'suffix': '%',
+            'scale': 'YlGn'
+        }
+
+        self.general_stats_addcols(percentages, headers)
+
+
+    def write_stats_to_file(self):
+        """Writes stats to data file."""
+        self.write_data_file(self.data, 'multiqc_disambiguate')
+
+
+    def add_stats_plot(self):
+        """Plots alignment stats as bargraph."""
+
+        keys = OrderedDict()
+        keys['species_a'] = {'color': '#437bb1', 'name': 'Species a'}
+        keys['species_b'] = {'color': '#b1084c', 'name': 'Species b'}
+        keys['ambiguous'] = {'color': '#333333', 'name': 'Ambiguous'}
+
+        self.add_section(
+            name='Percent ambiguous',
+            anchor='Percent ambiguous',
+            description='Disambiguation metrics, in number of reads.',
```

Comment:
Perhaps instead of this, supply a `config` dict to the `bargraph.plot` function below with an axis label and title (see [docs](http://multiqc.info/docs/#bar-graphs)). For example:

```python
pconfig = {
  'id': "disambiguate_alignments",
  'title': "Disambiguate Alignments",
  'cpswitch_counts_label': "# Reads",
  'ylab': "# Reads"
}
plot=bargraph.plot(self.data, keys, pconfig)
```